docs/install/any-kubernetes-cluster.md

```diff
@@ -587,13 +587,13 @@ To learn more about the Cloud Audit Logs source, try [our sample](../eventing/sa
 You can find a number of samples for Knative Eventing [here](../eventing/samples/README.md). A quick-start guide is available [here](../eventing/getting-started.md).
 
 
-## Installing the Monitoring bundle
+## Installing the Observability plugin
 
 {{< feature-state version="v0.2" state="alpha" >}}
 
-Knative provides a bundle of monitoring components that can be used to make the Serving and Eventing components more observable.
+Knative provides a collection of observability components that can be used to make the Serving and Eventing components more observable.
 
-Before you install any specific monitoring components, you must first install the core monitoring pieces:
+Before you install any specific observability components, you must first install the core:
 
 ```bash
 kubectl apply --filename {{< artifact repo="serving" file="monitoring-core.yaml" >}}
```

Comment:
Can we add below this:

"After the core is installed, you can choose to install one or all of the following observability plugins:"